index.html

```diff
@@ -59,7 +59,7 @@ <h1>Slideshow Test</h1>
     <div class="dcf-bleed dcf-wrapper dcf-pt-9 dcf-pb-9 unl-bg-lightest-gray">
       <div class="dcf-grid dcf-mb-6">
         <div class="dcf-col-100% dcf-col-33%-start@sm dcf-col-50%-start@lg">
-          <div class="dcf-slideshow dcf-slideshow-nojs" role="region" aria-label="slideshow" tabindex="0">
+          <div class="dcf-slideshow" role="region" aria-label="slideshow" tabindex="0">
```

Comment:
Not really a big deal but the `aria-label` for each slide show should be unique.  It would cause axe error in web audit.